www/include/configuration/configKnowledge/display-serviceTemplates.php

```diff
@@ -99,14 +105,15 @@
     $proc->setServiceInformations();
 
     // Get Services Template Informations
-    $query = "SELECT SQL_CALC_FOUND_ROWS service_description, service_id " .
-        "FROM service " .
-        "WHERE service_register = '0' " .
-        "AND service_locked = '0' ";
+    $query = "
+        SELECT SQL_CALC_FOUND_ROWS service_description, service_id
+            FROM service
+            WHERE service_register = '0'
+            AND service_locked = '0' ";
     if (!empty($postServiceTemplate)) {
         $query .= " AND service_description LIKE :postServiceTemplate ";
     }
-    $query .= "ORDER BY " . $orderby . " " . $order . " LIMIT " . $num * $limit . ", " . $limit;
+    $query .= "ORDER BY " . $orderBy . " " . $order . " LIMIT " . $num * $limit . ", " . $limit;
```

Comment:
could you bind offset and limit like you did here : https://github.com/centreon/centreon/pull/8815/commits/c812cb04e0454e113c5dde7368798265f11d7cc1#diff-e8d3d31415f23da2771e17637d4355f9R113